src/main/java/seedu/address/logic/Logic.java

```diff
@@ -16,10 +18,11 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
```

Comment:
Hmm... according to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments), seems like single space between exception name and reason might be the way to go. Should we just leave this line as it is?

Reply:
Might have been my IntelliJ auto formatter aligning it with above exceptions. I'll change the settings. Let's take note of this and sync our settings across other team members as well 👍

Reply:
Great, thanks!